contrib/win32/win32compat/inc/unistd.h

```diff
@@ -29,9 +29,13 @@ int w32_write(int fd, const void *buf, size_t max);
 int w32_writev(int fd, const struct iovec *iov, int iovcnt);
 
 int w32_isatty(int fd);
+#if _MSC_VER >= 1920  // VS 2019
+#define isatty w32_isatty
+#else
 /* can't do this #define isatty w32_isatty
-* as there is a variable in code named isatty*/
+ * as there is a variable in code named isatty*/
```

Comment:
Interesting, is this no more an issue with VS 2019 ? or "isatty" is no more being as a variable anywhere ?

Reply:
This issue do not appear on my VS2019 (16.2.4)  at least.

Reply:
I saw compiler error message says header file which was shipped by Windows SDK (maybe) contains a prototype of  `isatty()`.